src/components/Button/ActionButton.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from "react"
+import styled from "@emotion/styled"
+import { pxToRem } from "../ThemeProvider/typography"
+import {
+  ButtonRoot,
+  ButtonLinkRoot,
+  RESPONSIVE_SIZES,
+  DEFAULT_PROPS,
+} from "./Button"
+import type { ButtonStyleProps, ButtonSize } from "./Button"
+import type { LinkAdapterPropsOverrides } from "../LinkAdapter/LinkAdapter"
+
+type ActionButtonStyleProps = Omit<ButtonStyleProps, "startIcon" | "endIcon">
+type ActionButtonProps = ActionButtonStyleProps & React.ComponentProps<"button">
+
+const actionStyles = (size: ButtonSize) => {
+  return {
+    minWidth: "auto",
+    padding: 0,
+    height: {
+      small: "32px",
+      medium: "40px",
+      large: "48px",
+    }[size],
+    width: {
+      small: "32px",
+      medium: "40px",
+      large: "48px",
+    }[size],
+    "& svg, & .MuiSvgIcon-root": {
+      width: "1em",
+      height: "1em",
+      fontSize: pxToRem(
+        {
+          small: 20,
+          medium: 24,
+          large: 32,
+        }[size],
+      ),
+    },
+  }
+}
+/**
+ * A button that should contain a remixicon icon and nothing else.
+ * See [ActionButton docs](https://mitodl.github.io/smoot-design/?path=/docs/smoot-design-actionbutton--docs).
+ *
+ * See also:
+ * - [ActionButtonLink](https://mitodl.github.io/smoot-design/?path=/docs/smoot-design-actionbutton--docs#links)
+ * - [Button](https://mitodl.github.io/smoot-design/?path=/docs/smoot-design-button--docs) for text buttons
```

Comment:
Should these links be relative? 

- `[ActionButtonLink](#links)` for same page anchors (finding that the page doesn't scroll otherwise). Same issue in Button.tsx 

- `[Button](/docs/smoot-design-button--docs)`  (the docgen adds `/smoot-design/?path=` for us).

Reply:
I wish.

The main documentation is storybook comes from, in order of precedence:
1. Highest: The story JSDoc comment
2. The meta JSDoc comment
3. Lowest: The component's JSDoc comment

My hope is to use (3) in most cases, because (3) is also what shows up in IDE help text, e.g., 

![image](https://github.com/user-attachments/assets/e990386f-e255-46c1-900e-c331ac606606)

To be useful in IDEs, they need to be absolute links.

I don't know of any good way to make them relative in the source code but show up as absolute in the IDE.

----

 related: I believe that what MUI does for this is they all of their URLs are absolute to production, and their local dev setup replaces the production URLs to local runs when running in dev mode. Not worth our effort.